frontend/packages/erd-core/src/components/ERDRenderer/LeftPane/MenuItemLink/MenuItemLink.tsx

```diff
@@ -0,0 +1,33 @@
+import { SidebarMenuButton, SidebarMenuItem } from '@liam-hq/ui'
+import type { FC, ReactNode } from 'react'
+import styles from './MenuItemLink.module.css'
+
+export type Props = {
+  label: string
+  href: string
+  isExternalLink?: boolean
+  icon: ReactNode
+}
+
+export const MenuItemLink: FC<Props> = ({
+  label,
+  href,
+  isExternalLink,
+  icon,
+}) => {
+  return (
+    <SidebarMenuItem>
+      <SidebarMenuButton asChild>
+        <a
+          className={styles.link}
+          href={href}
+          target={isExternalLink ? '_blank' : '_self'}
+          rel={isExternalLink ? 'noreferrer' : ''}
```

Comment:
Considering self-hosting via CLI, it might be better to always use `noreferrer` regardless of `isExternalLink` ?  
That would also ensure consistency with `frontend/packages/erd-core/src/components/ERDRenderer/AppBar/AppBar.tsx` .

Reply:
I meant:

```suggestion
          rel={'noreferrer'}
```

If that looks good to you, I'd like to include it in the next PR!

Reply:
https://github.com/liam-hq/liam/pull/648

Reply:
I forgot to consider the cli version...
In the web version, the transition is within the same domain, so I want to be able to track the referral source in GA4 without adding `noreferrer`.
Therefore, I thought it would be a good idea to add props for whether to open in a separate tab and whether it is the same domain.

Reply:
Thank you!